dts/arm/st/l1/stm32l1-pinctrl.dtsi

```diff
@@ -39,6 +39,18 @@
 					tx = <STM32_PIN_PB10 (STM32_PINMUX_ALT_FUNC_7 | STM32_PUSHPULL_NOPULL)>;
 				};
 			};
+			usart3_pins_b: usart3_b {
```

Comment:
Talking about moving targets: #https://github.com/zephyrproject-rtos/zephyr/pull/19765

Reply:
Are you _serious_ ?! Are you actually at all interested in having outside contributors to this project?!  Your PR is newer and hasn't merged.  Updated however, I'm deep in the time sink, will be nice to get it finished.

Reply:
@karlp, I'm actually trying to make things easier to use for people (this is the reason for this PR).
And maybe it wasn't clear, but this wasn't a request to update your PR but a pure comment, as I expect this could indeed may take time to me merged.
I'm reviewing your PR several times a day and helping you to finding out issues that are preventing you to get CI passed (while I could wait until you get it sorted), so yes I consider I'm actually helping contributions.
Keep calm and go on (please).

Reply:
I know, I'm sorry I got a bit worked up on it, I have very much appreciated the attention you've been giving to this PR otherwise.  I've already dropped the pinctrl stuff though, do you want me to put it back in again?  It just depends who merges first.  I'm in no rush for it, it can easily be after your pinctrl stuff.

Reply:
No probs.
Current version is fine, this dts stuff is no op anyway. And since you removed it there is no dependency, we're fine.
Though, thinking about it, I'll have a last request for this PR, I should have asked it at the beginning, sorry (Cf upcoming comment).